tests/download_test_data.py

```diff
@@ -13,10 +15,11 @@ def download_file(url, local_filename):
                 f.write(chunk)
 
 
+raw_dir = "../nbs_tests/test_data"
 url_template = (
```

Comment:
I created a PR with the OOP test data downloader from alphaDIA.
I think this is something we could share.

Reply:
We are going to discuss this in alphaX meeting